spec/integration/create_subscriber_list_spec.rb

```diff
@@ -18,23 +18,9 @@
       create_subscriber_list
       subscriber_list = response_subscriber_list
 
-      expect(subscriber_list.keys.to_set.sort).to eq(
-        %w[
-          id
-          title
-          slug
-          document_type
-          created_at
-          updated_at
-          url
-          tags
-          links
-          email_document_supertype
-          government_document_supertype
-          active_subscriptions_count
-          tags_digest
-          links_digest
-        ].to_set.sort,
+      expect(subscriber_list.keys).to include(
+        "id", "title", "slug", "document_type",
```

Comment:
it doesn't _quite_ match the list you linked (url is expected here, but not there). If we want it to match exactly, would it be worth defining as a constant somewhere?

Reply:
Oh yes, thanks for checking. I've removed `url`, as it's also not used by any of the callers. The APIs are free to diverge, so I don't think it's worth introducing coupling between the tests at this stage. Rather, the existing test was just a helpful precedent to reduce the verbosity of this one.